dcos_test_utils/test_dcos_api.py

```diff
@@ -70,3 +70,9 @@ def test_dcos_client_api(mock_dcos_client):
     cluster.head('')
     cluster.patch('')
     cluster.options('')
+
+
+def test_jobs_property(mock_dcos_client):
```

Comment:
Is this enough?

Reply:
For now, yes. Adding integration tests in parallel. Once that lands then something like this module should require an integration test. Otherwise, we verify this by bumping dcos/dcos to point at this PR and use this code.

So, lets link this work with removing `metronome_one_off` from dcos-integration-test as this PR should be deprecating that function

Reply:
@mellenburg  - there will be integration tests and there should be unit-tests.

When a jobs.py is created, we should have coverage included with addition of unit-tests. Mocks can sometimes signify that they are unit-tests.

There is a scope for adding unittest with this change IMO.

Reply:
I personally think this is enough of a unit test but I could understand an argument for more coverage (e.g. all of the methods in the jobs module).

Integration test PR is here:
https://github.com/dcos/dcos-test-utils/pull/36